zanata-test-war/src/main/java/org/zanata/util/SampleProjectProfile.java

```diff
@@ -125,28 +126,41 @@ private void purgeLuceneIndexes() {
     }
 
     public void makeSampleLanguages() {
-        makeLanguage(LocaleId.FR);
+        makeLanguage(true, LocaleId.FR);
+
+        makeLanguage(true, new LocaleId("hi"));
 
-        makeLanguage(new LocaleId("hi"));
+        makeLanguage(true, new LocaleId("pl"));
+    }
 
-        makeLanguage(new LocaleId("pl"));
+    public HLocale makeLanguage(boolean enabledByDefault, LocaleId localeId,
+            String displayName, String nativeName) {
+        return forLocale(enabledByDefault, localeId, displayName, nativeName)
+                .makeAndPersist(entityManager,
+                    HLocale.class);
     }
 
-    public void makeLanguage(LocaleId localeId) {
-        forLocale(true, localeId).makeAndPersist(entityManager,
-                HLocale.class);
+    public HLocale makeLanguage(boolean enabledByDefault, LocaleId localeId) {
```

Comment:
It's generally good to add new parameters at the end of the list, rather than inserting them before the old ones.  Doesn't matter too much here though.